packages/sidecar/src/streaming.ts

```diff
@@ -0,0 +1,67 @@
+import type { Context } from "hono";
```

Comment:
Is this module copy/pasted from somewhere? If not why are we concerned about `bun` at all?